packages/app-builder/src/components/Scenario/AstBuilder/AstBuilderNode/AggregationEdit/Modal.tsx

```diff
@@ -3,14 +3,16 @@ import {
   type AggregationAstNode,
   aggregationAstNodeName,
   type AstNode,
-  computeValidationForNamedChildren,
-  type EvaluationError,
   NewConstantAstNode,
 } from '@app-builder/models';
 import {
   type AggregatorOperator,
   aggregatorOperators,
 } from '@app-builder/models/editable-operators';
+import {
+  computeValidationForNamedChildren,
+  type EvaluationError,
+} from '@app-builder/models/node-evaluation';
```

Comment:
There is a lot of noise like this (I try to move out of barel files so there are a lot of import changes)